_docs/Qabel-Protocol-Box.md

```diff
@@ -512,7 +515,7 @@ Remove a share to one or more contacts
 1. Remove the contacts' public keys from the share info of the folder in the index DM, set the device id and increment the version
 1. Download recursively all DM
 1. Upload an archive of all those DM with the name "backup_" + ref in the root directory (as a file object).
-1. Delete recursively all DM from the share (See [Delete Multiple Objects](https://docs.aws.amazon.com/AmazonS3/latest/API/multiobjectdeleteapi.html) )
+1. Let the block server recursively delete all DM from the share
```

Comment:
Shall I remove or mark all processes which are not yet implemented, so that they can be updated as soon as they are implemented?